.github/workflows/publish_plugin_to_hub.yml

```diff
@@ -0,0 +1,76 @@
+name: Publish plugin to hub
+on:
+  push:
+    tags:
+      - "plugins-*-*-v*.*.*"
+jobs:
+  publish-plugin-to-hub:
+    timeout-minutes: 60
+    runs-on: large-ubuntu-monorepo
+    steps:
+      - name: Split tag
+        id: split
+        run: |
+          tag=${{ github.ref_name }}
+          plugin_type=$(echo $tag | cut -d- -f2)
+          plugin_name=$(echo $tag | cut -d- -f3)
+          plugin_version=$(echo $tag | cut -d- -f4-)
+          echo "plugin_name=${plugin_name}" >> $GITHUB_OUTPUT
+          echo "plugin_type=${plugin_type}" >> $GITHUB_OUTPUT
+          echo "plugin_version=${plugin_version}" >> $GITHUB_OUTPUT
+          echo "plugin_dir=plugins/${plugin_type}/${plugin_name}" >> $GITHUB_OUTPUT
+
+      # Fail if not a valid SemVer string
+      - name: Parse semver string
+        uses: booxmedialtd/ws-action-parse-semver@7784200024d6b3fc01253e617ec0168daf603de3
+        id: semver_parser
+        with:
+          input_string: ${{ steps.split.outputs.plugin_version }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-1.20.5-publish-plugin-to-hub-cache-${{ hashFiles(format('{0}/{1}', steps.split.outputs.plugin_dir, 'go.sum')) }}
+          restore-keys: |
+            ${{ runner.os }}-go-1.20.5-publish-plugin-to-hub-cache-plugins-${{ steps.split.outputs.plugin_type }}-${{ steps.split.outputs.plugin_name }}
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: ${{ steps.split.outputs.plugin_dir }}/go.mod
+
+      - name: Get Release Notes
+        id: release-notes
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const { data } = await github.rest.repos.getReleaseByTag({
+              owner: "cloudquery",
+              repo: context.repo.repo,
+              tag: context.ref.replace('refs/tags/', ''),
+            });
+            return data.body;
+
+      - name: Run publish package command
+        working-directory: ${{ steps.split.outputs.plugin_dir }}
+        run: |
+          go run main.go package -m "${{ steps.release-notes.outputs.result }}" . ${{ steps.split.outputs.plugin_version }}
```

Comment:
Right now this will only work for Go plugins, are other plugins being excluded somehow?

Reply:
Ah right, I'll update it to ignore other plugins

Reply:
Fixed and based on the logic from https://github.com/cloudquery/cloudquery/blob/7d3aaa8950383cbaf5de5a016c9cd594bbbbc054/.github/workflows/release_plugin.yml#L48

I intentionally duplicated the file so not to interfere with existing publishing flow. We can merge them later on if needed